Makefile

```diff
@@ -16,7 +16,6 @@ compile:
 	@for dir in $(DIRS) ; do \
 		(cp README.md $(OUTDIR)/$(APPNAME)-$$dir/README.md) ;\
 		(cp LICENSE $(OUTDIR)/$(APPNAME)-$$dir/LICENSE) ;\
-		(cp sample-bchd.conf $(OUTDIR)/$(APPNAME)-$$dir/sample-bchd.conf) ;\
```

Comment:
I would keep this so the user has a reasonable config to look at. =)

Reply:
fixed in 73929c44